swap_meet/item.py

```diff
@@ -1,2 +1,52 @@
+import uuid
 class Item:
-    pass
\ No newline at end of file
+    """
+    A class that represents item.
+    Each Item have:
+    an attribute named id, which is an unique integer as default; 
+    an attribute named condition, which is an integer representing the item's condition, default value is 0 (the poorest);
+    an attribute named age, which is an integer representing the item's age, default value is 0.
+    """
+    def __init__(self, id=None, condition=0, age=0):
+        # when manually set id, check if it's an integer, if not: raise TypeError.
+        if id:
+            if isinstance(id, int):
+                self.id = id
+            else:
+                raise TypeError("Id must be an integer.")
+        # when id is not provided, generate an id with integer.
+        else:
+            self.id = uuid.uuid4().int
+        # If condition out of range, raise ValueError.    
+        if condition > 5:
+            raise ValueError("Condition value range from 0 to 5.")
```

Comment:
I really like your decision to raise errors in these cases. I wonder what the unit tests for these cases would look like 😄 😄 😄